go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/cosmos/gogoproto v1.4.10
 	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v7 v7.1.3
 	github.com/cosmos/ibc-go/v7 v7.3.2
-	github.com/cosmos/interchain-security/v4 v4.1.0
+	github.com/cosmos/interchain-security/v4 v4.1.0-rc0.0.20240403093553-272f32a9e2fd
```

Comment:
Will be replaced by `v4.1.0-lsm` once it's cut.

Reply:
Can you remove the `toolchain go1.22.0` line? I cannot make a comment to an unchanged line.